widgets/mini_player/public/style.css

```
@@ -0,0 +1,225 @@
+    width: 50%;
+    height: 50%;
+    color: var(--homey-color-mono-050);
+    flex-shrink: 0;
+}
+
+.homey-dark-mode .artwork-placeholder {
+    color: var(--homey-color-mono-100);
+}
+
+.artwork-placeholder path {
+    fill: currentColor;
```

Stylelint (17.4.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)